honeycombio/intro-to-o11y-go
# Updating the instructions to set the api key first as the app won't start without it set

## Which problem is this PR solving?

Updating the instructions to set the api key first as the app won't start without it set

## Short description of the changes

1. The Go app fails on startup if the API key isn't set, so tweaked the order of instructions to put setting the API key env variable ahead of starting the app
2. Adding additional verbiage and a screenshot for the Gitpod setup

## How to verify that this has the expected result

Run through the Gitpod instructions and ensure the app starts and data gets to Honeycomb
